### DISCUSSION TITLE
[Beginner Question] How can I verify that my pivpn installation is up to date?

### DISCUSSION THREAD

**Discussion by Outlet2048** [pivpn/pivpn#1759] [Support]
I have tried running pivpn -up and it tells me that I should use apt
I obviously update my system frequently but havent yet noticed any pivpn packages being updated

The output of the debug command shows the last commit being from the end of 2021, however this would be in line with what the support page says (if you installed PiVPN on or after March 17th 2020 WireGuard will be upgraded via the package manager (APT))

I have also used the install script and selected repair / update but both options didn't really help me in figuring out what version I'm currently running

After checking the releases page (https://github.com/pivpn/pivpn/releases) it seems like there are many updates so it's hard to believe i've missed them all when updating via apt

Therefore my question:
how can I verify what version of pivpn / wireguard protocol I'm running and whether or not it is the latest version?

**Comment by coolapso** [comment]
you don't upgrade pivpn. 

PiVPN is not a package.

All that is worth being updated is updated via package manager. 
PiVPN is simply an installer, and that's it. the Scripts, they might only need to be updated if you notice any bug, but if you don't then you don't need.

this might be worth moving into the docs page: https://github.com/pivpn/pivpn/wiki/Upgrading

**Reply by Outlet2048** [reply]
Hmm i see, thanks
If i understand correctly then I only have to bother with keeping wireguard itself up to date, correct?

Which seems to be the case: wireguard is already the newest version (1.0.20210223-1).
Even if that versions seems to be from 2021?

Thanks!
